examples/jsm/webxr/VRButton.js

```diff
@@ -176,7 +176,7 @@ class VRButton {
 
 	static registerSessionGrantedListener() {
 
-		if ( 'xr' in navigator ) {
+		if ( typeof navigator !== 'undefined' && 'xr' in navigator ) {
```

Comment:
Can you apply the same fix to `ARButton` and `XRButton`?

Reply:
`VRButton` is unique since `registerSessionGrantedListener` is called eagerly within the file.